src/components/DemoWrapper.js

```diff
@@ -11,50 +11,53 @@ import { SettingsContext } from './Settings/SettingsProvider'
 import { StorageContext } from './StorageContext'
 import Tabs from './Tabs'
 
-const styles = {
-  grid: css`
-    height: 100vh;
-    box-sizing: border-box;
-    max-height: 100vh;
-    min-height: 100vh;
-    display: flex;
-  `,
-  wrapper: css`
-    flex-grow: 2;
-    min-height: 0;
-    box-sizing: border-box;
-    height: 100vh;
-    max-height: 100vh;
-    overflow-y: scroll;
-    position: relative;
-  `,
-  main: css`
-    display: flex;
-    width: 100%;
-    height: 100%;
-    overflow-y: scroll;
-    min-height: 0;
-    min-width: 0;
-
-    & > .MuiCard-root {
-      overflow: scroll;
-    }
-  `,
-  contents: css`
-    overflow-y: scroll;
-    flex-grow: 1;
-    display: grid;
-    grid-template-rows: 1fr 1fr;
-    min-height: calc(100vh - 64px);
-  `,
-  appBar: css`
-    z-index: 3000;
-    position: relative;
-    & .MuiIconButton-root {
-      margin-right: 8px;
-    }
-  `,
-}
+const wrapperCss = css`
+  display: grid;
+  height: 100vh;
+  max-height: 100vh;
+  grid-template-rows: 48px 1fr;
+`
+
+const contentCss = css`
+  height: 100%;
+  display: flex;
+  background: #283048;  /* fallback for old browsers */
+background: -webkit-linear-gradient(to left, #859398, #283048);  /* Chrome 10-25, Safari 5.1-6 */
+background: linear-gradient(to left, #859398, #283048); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
```

Comment:
I thought Emotion was suppose to do all the _fallback_ stuff for us.

Reply:
I think it handles webkit stuff. We'll leave the fallback color for now.